.github/actions/e2e_sonobuoy/action.yml

```
@@ -45,7 +46,7 @@ runs:
       if: always() && !env.ACT
       uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
       with:
-        name: "sonobuoy_logs_${{ inputs.cloudProvider}}.tar.gz"
+        name: "sonobuoy_logs_${{ inputs.artifactNameSuffix }}.tar.gz"
```

```suggestion
        name: "${{ inputs.artifactNameSuffix }}_sonobuoy_logs.tar.gz"
```

No need for making the prefix a suffix? ^^

---

I'd prefer a suffix for filenames.